<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PThruster</class>
 <widget class="QWidget" name="PThruster">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>97</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QLabel" name="FwdRevT">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>460</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QScrollBar" name="FwdRevTSlider">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>320</y>
     <width>20</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>10</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QScrollBar" name="HeadingTSlider">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>20</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>10</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="PrtSrbT">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>300</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="UpDwnT">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>140</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>51</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="title">
    <string>Range</string>
   </property>
   <widget class="QRadioButton" name="UDTenth">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>.1</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="UDTen">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="UDOne">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>160</y>
     <width>51</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="title">
    <string>Range</string>
   </property>
   <widget class="QRadioButton" name="PSTenth">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>.1</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="PSTen">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="PSOne">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>320</y>
     <width>51</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="title">
    <string>Range</string>
   </property>
   <widget class="QRadioButton" name="FRTenth">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>.1</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="FRTen">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="FROne">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QScrollBar" name="MarkTSlider">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>0</y>
     <width>21</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="maximum">
    <number>10</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>HeadingTSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>PThruster</receiver>
   <slot>SetPrtSrbValue()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>79</x>
     <y>249</y>
    </hint>
    <hint type="destinationlabel">
     <x>49</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>FwdRevTSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>PThruster</receiver>
   <slot>SetFwdRevValue()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>79</x>
     <y>409</y>
    </hint>
    <hint type="destinationlabel">
     <x>49</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>UDOne</sender>
   <signal>toggled(bool)</signal>
   <receiver>PThruster</receiver>
   <slot>UpDwnRange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>25</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>42</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>UDTen</sender>
   <signal>toggled(bool)</signal>
   <receiver>PThruster</receiver>
   <slot>UpDwnRange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>25</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>42</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>UDTenth</sender>
   <signal>toggled(bool)</signal>
   <receiver>PThruster</receiver>
   <slot>UpDwnRange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>25</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>42</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PSOne</sender>
   <signal>toggled(bool)</signal>
   <receiver>PThruster</receiver>
   <slot>PrtSrbRange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>25</x>
     <y>218</y>
    </hint>
    <hint type="destinationlabel">
     <x>42</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PSTen</sender>
   <signal>toggled(bool)</signal>
   <receiver>PThruster</receiver>
   <slot>PrtSrbRange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>25</x>
     <y>248</y>
    </hint>
    <hint type="destinationlabel">
     <x>42</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PSTenth</sender>
   <signal>toggled(bool)</signal>
   <receiver>PThruster</receiver>
   <slot>PrtSrbRange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>25</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>42</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>FROne</sender>
   <signal>toggled(bool)</signal>
   <receiver>PThruster</receiver>
   <slot>FwdRevRange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>25</x>
     <y>378</y>
    </hint>
    <hint type="destinationlabel">
     <x>42</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>FRTen</sender>
   <signal>toggled(bool)</signal>
   <receiver>PThruster</receiver>
   <slot>FwdRevRange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>25</x>
     <y>408</y>
    </hint>
    <hint type="destinationlabel">
     <x>42</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>FRTenth</sender>
   <signal>toggled(bool)</signal>
   <receiver>PThruster</receiver>
   <slot>FwdRevRange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>25</x>
     <y>348</y>
    </hint>
    <hint type="destinationlabel">
     <x>42</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>FwdRevTSlider</sender>
   <signal>actionTriggered(int)</signal>
   <receiver>PThruster</receiver>
   <slot>SetFwdRevValue()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>380</y>
    </hint>
    <hint type="destinationlabel">
     <x>42</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>HeadingTSlider</sender>
   <signal>actionTriggered(int)</signal>
   <receiver>PThruster</receiver>
   <slot>SetPrtSrbValue()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>220</y>
    </hint>
    <hint type="destinationlabel">
     <x>42</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>MarkTSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>PThruster</receiver>
   <slot>SetUpDwnValue()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>42</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>MarkTSlider</sender>
   <signal>actionTriggered(int)</signal>
   <receiver>PThruster</receiver>
   <slot>SetUpDwnValue()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>42</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>Exit()</slot>
  <slot>SetUpDwnValue()</slot>
  <slot>SetPrtSrbValue()</slot>
  <slot>SetFwdRevValue()</slot>
  <slot>UpDwnRange()</slot>
  <slot>PrtSrbRange()</slot>
  <slot>FwdRevRange()</slot>
 </slots>
</ui>
